docs/topics/downloader-middleware.rst

```diff
@@ -1068,11 +1068,14 @@ Native implementation, provides better speed than Protego.
 
 In order to use this parser:
 
+.. warning:: Does not support Python 3.9+
+
 * Install `Reppy <https://github.com/seomoz/reppy/>`_ by running ``pip install reppy``
```

Comment:
It’s not exactly what I had in mind, but I guess it makes sense to keep the warning close to the installation instructions.

In that case, what about nesting it within the installation step?

```suggestion
* Install `Reppy <https://github.com/seomoz/reppy/>`_ by running ``pip install reppy``

  .. warning:: Does not support Python 3.9+
```

Reply:
seems good ....